source/plugins/ruby/CustomMetricsUtils.rb

```diff
@@ -13,6 +13,10 @@ def check_custom_metrics_availability
             if aks_region.to_s.empty? || aks_resource_id.to_s.empty?
                 return false # This will also take care of AKS-Engine Scenario. AKS_REGION/AKS_RESOURCE_ID is not set for AKS-Engine. Only ACS_RESOURCE_NAME is set
             end
+            is_ArcA_Cluster = ENV['IS_ARCA_CLUSTER']
+            if is_ArcA_Cluster.to_s.downcase == "true" && ENV['ARCA_Metrics_Endpoint']
```

Comment:
The endpoint can be checked by `.to_s.empty?` as well
```suggestion
            if is_ArcA_Cluster.to_s.downcase == "true" && ENV['ARCA_Metrics_Endpoint'].to_s.empty?
```